bootstrap.php

```diff
@@ -79,6 +80,8 @@ function ( $value ) {
 		'newfold_container_set',
 		function ( Container $container ) {
 
+			NFD_DATA_MODULE_VERSION === '2.4.21' && nfd_update_options_table( $container );
```

Comment:
Did the upgrade routine approach not work? That is the preferred method.

Reply:
we are getting this error - Uncaught NewfoldLabs\Container\NotFoundException : No entry was found for "plugin" identifier. 

As we talked about earlier, we're employing a hook call and specifying a version condition.

Reply:
@manikantakailasa, I was thinking we would add the hook to the upgrade routine file. So in the `2.4.22.php` file, add the `newfold_container_set` callback and move the code out of the called function into the hook.